src/mqt/bench/utils.py

```diff
@@ -428,3 +420,27 @@ def calc_supermarq_features(
         parallelism,
         liveness,
     )
+
+
+def get_benchmarks_module_paths_dict() -> dict[str, str]:
+    return {
+        "ae": "mqt.bench.benchmarks.ae",
+        "dj": "mqt.bench.benchmarks.dj",
+        "grover": "mqt.bench.benchmarks.grover",
+        "ghz": "mqt.bench.benchmarks.ghz",
+        "graphstate": "mqt.bench.benchmarks.graphstate",
+        "portfolioqaoa": "mqt.bench.benchmarks.qiskit_application_finance.portfolioqaoa",
+        "portfoliovqe": "mqt.bench.benchmarks.qiskit_application_finance.portfoliovqe",
+        "qaoa": "mqt.bench.benchmarks.qaoa",
+        "qft": "mqt.bench.benchmarks.qft",
+        "qftentangled": "mqt.bench.benchmarks.qftentangled",
+        "qgan": "mqt.bench.benchmarks.qiskit_application_ml.qgan",
+        "qpeexact": "mqt.bench.benchmarks.qpeexact",
+        "qpeinexact": "mqt.bench.benchmarks.qpeinexact",
+        "qwalk": "mqt.bench.benchmarks.qwalk",
+        "realamprandom": "mqt.bench.benchmarks.realamprandom",
+        "su2random": "mqt.bench.benchmarks.su2random",
+        "twolocalrandom": "mqt.bench.benchmarks.twolocalrandom",
+        "vqe": "mqt.bench.benchmarks.vqe",
+        "wstate": "mqt.bench.benchmarks.wstate",
+    }
```

Comment:
Looking at this, I believe this hard-coding can be eliminated.
All benchmarks are importable from `mqt.bench.benchmarks.<name>` (even the application-specific benchmarks, due to my recent addition of the `__init__.py` file).
Thus, I believe the methods that use this method here can be refactored to just use the above scheme.

Reply:
I changed this accordingly, at least for all benchmarks in `mqt/bench/benchmarks`. For the other benchmarks I could not get it to work since `importlib.find_module` always returned a `module not found` although the respective other application modules are explicitly re-exported in the `__init__`.